app/controllers/houses_controller.rb

```diff
@@ -0,0 +1,34 @@
+class HousesController < ApplicationController
+  def index
+    @house = House.all
+    render json: @house
+  end
+
+  def show
+    @house = House.where(id: params[:id]).first
+    render json: @house, status: :ok
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @house = @user.houses.create(house_params)
+    if @house.save
+      render json: @house, status: :created
+    else
+      render json: @house.errors, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @house = House.where(id: params[:id]).first
```

Comment:
- [ ] Using `find` here should be a better option then `where`, since you already have the `id`. This way you won't even need to use `first`.